packages/zowe-explorer/__tests__/__unit__/configuration/Profiles.unit.test.ts

```diff
@@ -120,6 +122,14 @@ function createGlobalMocks(): { [key: string]: any } {
         configurable: true,
     });
 
+    Object.defineProperty(SettingsConfig, "getDirectValue", {
+        value: createGetConfigMock({
+            "zowe.ds.default.sort": Sorting.DatasetSortOpts.Name,
+            "zowe.jobs.default.sort": Sorting.JobSortOpts.Id,
+        }),
```

Comment:
@Buckminsterfullerene02 Thanks for your contribution!

Took a look at the failing tests in this file, adding 1 more setting to the config mock should fix them 🙂 
```suggestion
        value: createGetConfigMock({
            "zowe.ds.default.sort": Sorting.DatasetSortOpts.Name,
            "zowe.jobs.default.sort": Sorting.JobSortOpts.Id,
            [Constants.SETTINGS_SECURE_CREDENTIALS_ENABLED]: false,
        }),
```